server/review.test.js

```diff
@@ -0,0 +1,51 @@
+const request = require('supertest')
+  , {expect} = require('chai')
+  , db = require('APP/db')
+  , app = require('./start')
+
+/* global describe it before afterEach */
+
```

Comment:
Merged. This comment will be added to Chloe's current branch dealing with these tests.